### DISCUSSION TITLE
sending diagnostic report

### DISCUSSION THREAD

**Discussion by cdhiraj40** [kiwix/kiwix-android#2753] [Q&A] [answered]
Shouldn't diagnostic reports be sent via direct mail to kiwix? the same way feedback works

![Screenshot_2021-12-03-16-50-12-14_c31b32364ce19ca8fcd150a417ecce58.jpg](https://user-images.githubusercontent.com/75211982/144594255-537de4db-dd98-410e-8990-ddb27c512e75.jpg)

**Comment by gouri-panda** [comment]
@cdhiraj40 Do you mean only email?

**Reply by cdhiraj40** [reply]
@gouri-panda yes

**Comment by gouri-panda** [comment, marked as answer]
It's good if we can share logs in multiple ways.
